pkg/ignition/builder.go

```diff
@@ -129,6 +131,17 @@ func (b *ignitionBuilder) GenerateConfig() (config ignition_config_types_32.Conf
 		0644, false,
 		[]byte("[connection]\nipv6.dhcp-duid=ll\nipv6.dhcp-iaid=mac")))
 
+	if b.additionalNTPServers != "" {
+	    chronyConfig := []byte{}
```

Comment:
i wonder if there is need for a default ntp server e.g. similar to the chrony conf file built here
https://github.com/openshift/installer/blob/d61300fbc2107236d85ecb7e60bd1d2fabc8ed51/pkg/asset/imagebased/configimage/clusterconfiguration.go#L22

Reply:
Docs say yes: https://docs.openshift.com/container-platform/4.16/installing/install_config/installing-customizing.html#installation-special-config-chrony_installing-customizing
I believe this will overwrite the default chrony.conf, so we have to include all of the default configuration options, as well as our additional options.

Reply:
I now append to the config file